qsh/__init__.py

```diff
@@ -172,15 +172,15 @@ def __init__(self, order_type, order_id, price, amount_rest):
     def __str__(self):
         return ";".join((str(self.type), str(self.id), str(self.price), str(self.amount_rest)))
 
-class QshFile:
+class QshFileHeader:
+    signature = "QScalp History Data"
+    version = None
+    application = None
+    comment = None
+    created_at = None
+    streams_count = None
```

Comment:
minor: This formatting of assignments ease refactoring. You don't need to change many lines on renaming of one attribute